src/osd/OSD.cc

```diff
@@ -352,12 +352,18 @@ void OSDService::identify_splits_and_merges(
   set<pair<spg_t,epoch_t>> *split_children,
```

Comment:
>  OSD: during test start, not all osds started due to consum map hang 

I think there is a typo in first's commit title: `consum map` -> `consume_map`.

Reply:
Done